setup.cfg

```diff
@@ -0,0 +1,56 @@
+[metadata]
+name = create-aio-app
+version = attr: create_aio_app.utils.versiontools.get_self_version
+description = The tool that helps quickly create a basis for aiohttp application
+long_description = file: README.md
+long_description_content_type = text/markdown
+platforms = 
+    POSIX
+author = Mykhailo Havelia
+author_email = misha.gavela@gmail.com
+license = MIT
+license_file = LICENSE
+url = https://github.com/aio-libs/create-aio-app
+project_urls = 
+    Source Code = https://github.com/aio-libs/create-aio-app
+    GitHub Issues = https://github.com/aio-libs/create-aio-app/issues
+    Documentation = https://create-aio-app.readthedocs.io/
+    CI: Travis = https://travis-ci.com/aio-libs/create-aio-app
+
+classifiers =
+    License :: OSI Approved :: MIT License
+    Intended Audience :: Developers
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Operating System :: POSIX
+    Development Status :: 4 - Beta
+    Framework :: AsyncIO
+
+keywords =
+    create-aio-app
+    cookiecutter
+    aiohttp
+
+
+[options]
+packages = find_namespace:
+include_package_data = True
```

Comment:
Interesting fact.

Some time ago, when we setup this tool we used `package_data` (files without `.py` prefix didn't copy ). But now all work pretty good. @webknjaz do u know something about it? Was this a bug in setuptools?